R/readIIASA_subs_taxes.R

```diff
@@ -9,7 +9,6 @@
 #' pre final energy category \item \code{subsidies_bulk}: subsidy rate per final
 #' energy category \item \code{energy}: final energy quantities per category }
 #' @return magpie object of the IIASA_subs_taxes data
-#' @author Christoph Bertram
```

Comment:
WHy did you remove the Author here? This is useful and important info, even if the person is no longer at PIK.

Reply:
I apologize. I thought the consensus was that we should avoid using author tags and instead rely on git blame. Will be but back.

Reply:
We have users wanting to track sources, but do not know how to use git blame (because they are not coder themselves). Besides, people who added or changed code are not always the same as the person to contact with questions. Lavinia is listed in git blame for code that has not been changed since the migration to GitHub, so that does not help. Other legacy functions have been updated lately only by people how did formatting or refactorings.

Reply:
Ok, but then surely it would be better to have actual contact people listed as authors, instead of people who don't work for PIK anymore no? (See #640 for the return of the authors tag.)